modules/apps/digital-signature/digital-signature-test/src/testFunctional/tests/DigitalSignatureDM.testcase

```diff
@@ -38,118 +44,110 @@ definition {
 
-		DMDocument.addCP(
-			dmDocumentDescription = "DM Document Description",
-			dmDocumentFile = "Alice's Adventures in Wonderland.pdf",
-			dmDocumentTitle = "DM Document PDF");
-
-		var dmDocumentTitle = "DM Document PDF";
+			var dmDocumentTitle = "DM Document PDF";
+		}
 
-		Click.waitForMenuToggleJSClick(
-			key_rowEntry = ${dmDocumentTitle},
-			locator1 = "Icon#ROW_VERTICAL_ELLIPSIS");
+		task ("When the user goes to digital signature by ellipises") {
+			DigitalSignature.goToDigitalSignatureEllipsis(dmDocumentTitle = "DM Document PDF");
+		}
 
-		AssertElementPresent(locator1 = "DigitalSignatureDM#DIGITAL_SIGNATURE_ELLIPSIS_COLLECT_BUTTON");
+		task ("Then confirm that you are on the page and delete the file") {
+			DigitalSignature.assertDigitalSignaturePage(assertPage = "New Digital Signature Envelope");
 
-		DMNavigator.openDocumentsAndMediaAdmin(siteURLKey = "guest");
+			DMNavigator.openDocumentsAndMediaAdmin(siteURLKey = "guest");
 
-		DMDocument.deleteCP(dmDocumentTitle = "DM Document PDF");
+			DMDocument.deleteCP(dmDocumentTitle = "DM Document PDF");
+		}
 	}
 
-	@description = "This is a test for LPS-135089. Verify if its possible get multiple signatures from diferent documents in DM"
+	@description = "LPS-135089. Verify if its possible get multiple signatures from diferent documents in DM"
 	@priority = 5
 	test CanGetMultipleSignaturesByButton {
-		property solutions.acceptance = "true";
-
-		DigitalSignature.enableDigitalSignature(siteStrategy = "Always Inherit");
+		task ("Given that user Test creates multiple documents") {
+			property solutions.acceptance = "true";
 
-		DMNavigator.openDocumentsAndMediaAdmin(siteURLKey = "guest");
-
-		DMDocument.addMultipleCP(dmDocumentFileList = "Document_1.doc,Document_2.jpg");
-
-		Click(
-			dmDocumentTitle = "Document_1",
-			locator1 = "DigitalSignatureDM#DIGITAL_SIGNATURE_SELECT_DOCUMENT");
-
-		Click(
-			dmDocumentTitle = "Document_2",
-			locator1 = "DigitalSignatureDM#DIGITAL_SIGNATURE_SELECT_DOCUMENT");
+			DMDocument.addMultipleCP(dmDocumentFileList = "Document_1.doc,Document_2.jpg");
+		}
 
-		Click.javaScriptClick(locator1 = "DigitalSignatureDM#DIGITAL_SIGNATURE_COLLECT_BUTTON");
+		task ("When the user goes to digital signature") {
+			DigitalSignature.documentDigitalSignature(documentlist = "Document_1,Document_2");
+		}
 
-		AssertElementPresent(locator1 = "DigitalSignatureDM#DIGITAL_SIGNATURE_SEND_BUTTON");
+		task ("Then confirm that you are on the page and delete the files") {
+			DigitalSignature.assertDigitalSignaturePage(assertPage = "New Digital Signature Envelope");
 
-		DMNavigator.openDocumentsAndMediaAdmin(siteURLKey = "guest");
+			DMNavigator.openDocumentsAndMediaAdmin(siteURLKey = "guest");
 
-		DMDocument.deleteCP(dmDocumentTitle = "Document_1");
+			DMDocument.deleteCP(dmDocumentTitle = "Document_1");
 
-		DMDocument.deleteCP(dmDocumentTitle = "Document_2");
+			DMDocument.deleteCP(dmDocumentTitle = "Document_2");
+		}
```

Comment:
As I sad above you can remove these delete steps due to they will be added on teardown